windows/src/engine/kmcomapi/processes/keyboard/kpinstallkeyboardlanguageprofiles.pas

```diff
@@ -340,22 +338,15 @@ function TKPInstallKeyboardLanguageProfiles.RegisterLocale(KeyboardName, BCP47Ta
 
 function TKPInstallKeyboardLanguageProfiles.ExtractBaseBCP47Tag(BCP47Tag: string): string;
 var
-  FScripts: TStringList;
   FTag: TBCP47Tag;
 begin
   Result := BCP47Tag;
 
-  FScripts := TStringList.Create;
   FTag := TBCP47Tag.Create(BCP47Tag);
   try
-    FScripts.Text := SuppressScriptSubtagRegistry;
-    if SameText(FTag.Script, FScripts.Values[FTag.Language]) then
-    begin
-      FTag.Script := '';
-      Result := FTag.Tag;
-    end;
+    FTag.Canonicalize;
+    Result := FTag.Tag;
```

Comment:
What happens if `FTag.Canonicalize` returns `''`?

Reply:
Okay, testing return value now during the install.